step.yml

```diff
@@ -120,9 +120,10 @@ inputs:
   - app_password: ""
     opts:
       title: "Apple ID: Application-specific password"
-      summary: Required if using Apple ID
+      summary: Optional if using Apple ID
       description: |-
-        An application-specific password for the Apple ID.
+        This input needs to be used if TFA enabled on the Apple ID, but no app-specific password added to the used Bitrise Apple ID connection.
```

Comment:
Use this input if TFA is enabled on the Apple ID but no app-specific password has been added to the used Bitrise Apple ID connection.